### ISSUE TITLE
Find out how "Setting.transfer_wait_time" setting is used

### ISSUE THREAD

**Issue by artur-beljajev** [internetee/registry#675]

**Comment by vohmar** [comment]
https://tools.ietf.org/html/rfc5730#section-2.9.3.4

EPP standard provides a  approval/rejection based mechanism to manage domain transfers requests. The default process puts transfer request to pending status requiring the current registrar to approve or reject the request using op attribute with value approve or reject. It is also possible to set time limit for the pending request and set the automatic action for when the time runs out. This is where the Setting.transfer_wait_time comes into play. In EIS we use setting 0 seconds of wait time and automatically approving the request with correct auhtinfo password presents.

To recap -Setting.transfer_wait_time sets the time for pendingTransfer status to wait for approval of current sponsoring registrar. When this time has passed automatic action must occur to either approve or reject the pending request.

**Comment by artur-beljajev** [comment]
> When this time has passed, automatic action must occur to either approve or reject the pending request.

If you mean some kind of CRON, then currently there is none.
